generate.py

```diff
@@ -150,47 +253,75 @@ def generate_message(media, target, conf, result):
         help='Debug level',
         default='INFO')
     parser.add_argument('dest', default='dist', type=writeable_dir)
+    parser.add_argument(
+        '-m',
+        '--minify',
+        dest='minify',
+        default=False,
+        action='store_true')
 
     args = parser.parse_args()
 
     log_level = getattr(logging, args.log_level)
     logging.basicConfig(level=log_level)
 
-    config = yaml.load(args.config)
+    config = yaml.full_load(args.config) # https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation
```

Comment:
Do we need full_load or would safe_load be sufficient?

Reply:
I'm not sure. The docs says the following for save_load:

```
Loads a subset of the YAML language, safely. This is recommended for loading untrusted input.
```

I don't know what subset they are speaking about and the input is not untrusted. For FullLoader the cods says that it's the default behaviour after issuing the warning I tried to get rid of.

Reply:
I would use the `safe_load` here since that doesn't allow nonsense like loading abitrary Python objects from YAML code (allowing code executionong). We do not need that and shouldn't allow that.